templates/service-template.yml

```diff
@@ -480,7 +475,7 @@ parameters:
 - name: OBSERVABILITY_CONFIG_REPO
   displayName: Observability configuration repo URL
   description: URL of the observability configuration repo
-  value: "https://api.github.com/repos/bf2fc6cc711aee1a0c2a/observability-resources-mk/contents"
+  value: "quay.io/rhoas/observability-resources-mk"
```

Comment:
What is the reason this has been changed? The new URL seems a quay container image repository while the previous one seems  GitHub content URL